internal/cmd/pipeline/command_create.go

```diff
@@ -65,9 +66,17 @@ func (c *command) create(cmd *cobra.Command, _ []string) error {
 	}
 
 	// validate sr id
+	srId := ""
 	srCluster, err := c.Config.Context().SchemaRegistryCluster(cmd)
 	if err != nil {
-		return err
+		if !strings.Contains(err.Error(), "Schema Registry not enabled") {
+			// ignore if the SR is not enabled
+			return err
+		}
+	}
+
+	if srCluster != nil {
+		srId = srCluster.Id
```

Comment:
Is there a `srCluster.GetId()` function we can call here to handle the `nil` check for us?